src/components/atoms/FormItem/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export interface formItemProps {
+  errorMessage?: string;
+  label?: string;
+  invalid?: boolean;
+  children: React.ReactNode;
+  className?: string;
+  htmlFor?: string;
+}
+
+const FormItem: React.FC<formItemProps> = ({
+  label,
+  invalid,
+  htmlFor,
+  errorMessage,
+  className,
+  children,
+}) => {
+  return (
+    <form>
```

Comment:
Invalid is a boolean prop. 

use this to change color of border to red: 

example: 


```suggestion
    <div className={invalid && 'border-rose-500'}>
    
```